cmd/flow-dps-indexer/README.md

```diff
@@ -3,7 +3,7 @@
 ## Description
 
 The Flow DPS Indexer binary implements the core functionality to create the index for past sporks.
-It needs a reference to the protocol state database of the spork, as well as the trie directory and an execution state checkpoint.
+It needs a reference to the protocol state database of the spork, as well as the trie directory, and an execution state checkpoint.
```

Comment:
This comma is actually wrong: https://www.grammarly.com/blog/comma-before-and/

Reply:
Nice catch